robottelo/hosts.py

```diff
@@ -1754,6 +1754,19 @@ def cli(self):
         self._cli._configured = True
         return self._cli
 
+    def install_satellite_or_capsule_package(self):
+        """Install Satellite/Capsule package.
+        Note: Make sure required repos are enabled before using this.
+        """
+        if settings.server.version.rhel_version != 9:
```

Comment:
@Gauravtalreja1 @Griffin-Sullivan should we split this too? like enabling module and then performing package install? or that'd be too much😄

Reply:
yes, it'll be too much as its just one line which will be unnecessary anyway in future